### PR TITLE
[Login] Fetch profile info and images at launching

### PR DESCRIPTION
**Pull request by duckmoll**
- Fetch image and info when logged in
- Fetch image and info when phone verified

Video in Slack.

### CODE REVIEW

File: NewDawn/NewDawn/ViewControllers/LoginViewControllers/PhoneAuthenticateViewController.swift [hunk 1]
```diff
@@ -133,11 +134,36 @@ class PhoneAuthenticateViewController: UIViewController {
...
+                        queue.tasks +=~ {
+                            LoginUserUtil.downloadOverwriteLocalImages(profile: user_profile!)
+                        }
+                        queue.tasks +=! {
+                            let mainPageStoryboard:UIStoryboard = UIStoryboard(name: "MainPage", bundle: nil)
+                            let homePage = mainPageStoryboard.instantiateViewController(withIdentifier: "MainTabViewController") as! MainPageTabBarViewController
+                            let appDelegate = UIApplication.shared.delegate
+                            appDelegate?.window??.rootViewController = homePage
+                        }
+                        queue.run()
```
Comment (tangziyi001) [inline NewDawn/NewDawn/ViewControllers/LoginViewControllers/PhoneAuthenticateViewController.swift]: Damn this is super awesome. Is TaskQueue() a built-in in Swift?

Comment (duckmoll) [reply NewDawn/NewDawn/ViewControllers/LoginViewControllers/PhoneAuthenticateViewController.swift]: HAHAHAH. No, I thought you installed this library.

Comment (tangziyi001) [reply NewDawn/NewDawn/ViewControllers/LoginViewControllers/PhoneAuthenticateViewController.swift]: Haha guess this was installed as a dependency of other libraries.
How did you find this though?

Comment (duckmoll) [reply NewDawn/NewDawn/ViewControllers/LoginViewControllers/PhoneAuthenticateViewController.swift]: When I was installing MixPanel, I found TaskQueue in the "frameworks" folder, so I looked it up. It turns out to be a convenient library to use.

**Pull request merged**